packages/storybook/src/nimble/spinner/spinner.stories.ts

```diff
@@ -53,7 +53,7 @@ const metadata: Meta<SpinnerArgs> = {
                 Requires 64px minimum margins.
                 </li></ul></details>`,
             options: Object.keys(spinnerSize),
-            table: { defaultValue: { summary: 'Small (16x16)' } },
+            table: { category: apiCategory.styles },
```

Comment:
Probably should capitalize "size" (in the Name column) to help reinforce that this isn't a symbol. We do the same thing for "Localizable labels".

Reply:
Capitalized and added more words to make it more of a sentence (here and in a couple other places).